packages/protocol/src/action-protocol/element-selection.ts

```diff
@@ -45,14 +51,32 @@ export namespace SelectAction {
         return Action.hasKind(object, KIND) && hasArrayProp(object, 'selectedElementsIDs') && hasArrayProp(object, 'deselectedElementsIDs');
     }
 
-    export function create(options: { selectedElementsIDs?: string[]; deselectedElementsIDs?: string[] } = {}): SelectAction {
+    export function create(options: { selectedElementsIDs?: string[]; deselectedElementsIDs?: string[] | boolean } = {}): SelectAction {
+        const { deselectedElementsIDs, ...remainingOptions } = options;
         return {
             kind: KIND,
-            selectedElementsIDs: [],
```

Comment:
I don't think that the spreading into remaining options is necessary here. We already use `options.selectedElementIds` and `options.deselectedElementIds` so there are no more remaining options 😉